docs/components/concepts/process-instance-creation.md

```diff
@@ -123,22 +110,29 @@ Start instructions have the same [limitations as process instance modification](
 Start instructions are supported for both `CreateProcessInstance` commands.
 
 <details>
-  <summary>Code example</summary>
-  <p>
-Create a process instance starting before the 'ship_parcel' element:
-
-```java
-client.newCreateInstanceCommand()
-  .bpmnProcessId("order-process")
-  .latestVersion()
-  .variables(Map.of("orderId", "1234"))
-  .startBeforeElement("ship_parcel")
-  .send()
-  .join();
+   <summary>Create a process instance and await results via Camunda 8 REST API</summary>
+   <p>
+
+```
+curl -L 'http://localhost:8080/v2/process-instances' \
+-H 'Content-Type: application/json' \
+-H 'Accept: application/json' \
+-d '{
+  "processDefinitionId": "order-process”,
+  "processDefinitionVersion": -1,
+  "startInstructions": [
+    {
+      "elementId": "ship_parcel"
+    }
+  ],
+  "variables": { "orderId": "1234" }
+}'
 ```
```

Comment:
This was the only example on the page that used a Java code example, while the rest used zbctl (which I converted to cURL). For consistency, I converted this one as well.

Reply:
Enthusiastic emoji response, resolving.